topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html

```diff
@@ -36,7 +36,7 @@
 contributors:
   - mtekman
   - astrovsky01
-
+  - nomadscientist
```

Comment:
Put your name before mine I think, this is a drastic and great improvement to these slides, so credit where credit is due ;-)